GoToProject/src/library/methods/firebaseDetailsSite.js

```diff
@@ -0,0 +1,12 @@
+import firestore from '@react-native-firebase/firestore';
+export const detailsSite = (idSite, setDataDetails) => {
+  firestore()
+    .collection('sites')
+    .doc(idSite)
+    .get()
+    .then(resourceSites => {
+      console.log(resourceSites._data);
```

Comment:
Recommendation: Do not upload the console.log () to GitHub, instead use the Alert element.

Reply:
done, thanks